openshift/job-kebechet-template.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: kebechet
+  annotations:
+    description: This is Kebechet Job Config, this template is meant to be used by Bots, but could also be used by humans...
+    openshift.io/display-name: Kebechet Job
+    version: 1.0.0
+    tags: poc,thoth,ai-stacks,kebechet
+    template.openshift.io/documentation-url: https://github.com/thoth-station/kebechet/
+    template.openshift.io/long-description: This is Kebechet used for automatic package releases and updates.
+    template.openshift.io/provider-display-name: Red Hat, Inc.
+  labels:
+    template: kebechet
+    app: kebechet
+
+parameters:
+  - description: The git token for the specific service
+    displayName: git Service Access Token
+    required: true
+    name: GIT_ACCESS_TOKEN
+  - description: The repositories full URL 
+    displayName: Repository URL
+    required: true
+    name: REPO_URL
+  - description: The name of the git service the repository lives on
+    displayName: git Service Name
+    required: true
+    name: SERVICE_NAME
+  - description: The ID of the Kebechet job
+    displayName: Job ID
+    required: true
+    name: KEBECHET_JOB_ID
+  - description: Determines if kebechet is in verbose mode
+    displayName: Verbose mode
+    required: true
+    name: KEBECHET_VERBOSE
+
+objects:
+- apiVersion: v1
+  kind: Job
```

Comment:
This does not look like a correct job template to me. This looks more like a cronjob template.

Reply:
Not sure if what you mean, I tried my best to follow the advise-job template